docs/release-notes/index.md

```diff
@@ -29,6 +29,15 @@ This is the list of changes to Taipy releases as they were published.
 [`taipy-templates` 4.0](https://pypi.org/project/taipy-templates/4.0.0/), and
 [`taipy-rest` 4.0](https://pypi.org/project/taipy-rest/4.0.0/) packages.
 
+## Known issues
+
+<h4><strong><code>taipy-gui</code></strong> 4.0.0</h4>
+
+- The [**-H**](../userman/advanced_features/configuration/gui-config.md#p-port) short command line
+  option for setting the server hostname is broken.<br/>
+  Please use the full **--host** option instead.<br/>
+  This issue will be fixed in the next technical release for Taipy GUI.
+
```

Comment:
Do you want to expose that on `develop` branch?
It makes sense to me to merge it in `release/4.0`, but it's strange to expose it on `develop` branch.

Reply:
Yes I do.
We do have a bad known issue in 4.0 (therefore it once was on develop).
This will be fixed in 4.0.1 (therefore mentioned as a 'Significant bug fixes' in the Release Notes, both in develop and the 4.0 branch).